src/CoreWCF.Primitives/src/CoreWCF/ServiceAuthorizationManager.cs

```diff
@@ -39,13 +77,33 @@ public virtual bool CheckAccess(OperationContext operationContext)
             operationContext.IncomingMessageProperties.Security.ServiceSecurityContext =
                 new ServiceSecurityContext(authorizationPolicies ?? EmptyReadOnlyCollection<IAuthorizationPolicy>.Instance);
 
-            // 3) Call the CheckAccessCore
+            // 3) Call the CheckAccessCoreAsync(OperationContext operationContext)
             return CheckAccessCore(operationContext);
```

Comment:
I think this comment should say CheckAccessCore

Reply:
Good catch! thanks!